napari/utils/shortcuts.py

```diff
@@ -73,6 +73,23 @@
     ],
     'napari:activate_image_pan_zoom_mode': [KeyCode.Digit2],
     'napari:activate_image_transform_mode': [KeyCode.Digit1],
+    'napari:image_hold_to_pan_zoom': [KeyCode.Space],
+    'napari:labels_hold_to_pan_zoom': [KeyCode.Space],
+    'napari:points_hold_to_pan_zoom': [KeyCode.Space],
+    'napari:shapes_hold_to_pan_zoom': [KeyCode.Space],
```

Comment:
These need unique names so you could bind a different key for each layer type. In app-model we could just use `napari:image:hold_to_pan_zoom`.